src/models/shipment.ts

```diff
@@ -0,0 +1,58 @@
+import {
+  Model,
+  Column,
+  Table,
+  CreatedAt,
+  UpdatedAt,
+  DataType,
+  ForeignKey,
+  BelongsTo,
+} from 'sequelize-typescript'
+
+import { ShipmentStatus, ShippingRoute } from '../server-internal-types'
+import Group from './group'
+
+@Table({
+  timestamps: true,
+})
+export default class Shipment extends Model {
+  @Column(DataType.STRING)
+  public shippingRoute: ShippingRoute
```

Comment:
Without the explicit `STRING` type here, sequelize complains becuase `ShippingRoute` is a TS enum. If someone knows a better way around this let me know!

Reply:
Sounds reasonable to me. I prefer strings for representing enums in DB columns since they're easier to read and decipher what a query is doing or what's in a row when accessing the DB directly.